R/frames.R

```diff
@@ -65,9 +75,49 @@ frame_1<- function(map,
 
 
   server <- function(input, output) {
-    output$map <- renderLeaflet({
-      map
-    })
+    output$img <- renderImage({
+
+      # create a tempdir
+      myDir = tempdir()
+
+      # save map as png file
+      mapshot(map,file = paste0(myDir,"map.png"))

```

Comment:
What I'm curious about here is that if we're able to have the mask without having to freeze the map with mapshot. 

The `save_map_svg` function gives us a frozen image of the map. 

If its possible to have the map freely move while being covered by the mask that would be ideal. 

This will be useful further down the line when implementing this code in a shiny app. 

Let me know what you think

Reply:
Never realised the map was interactive in the original setup. Good idea though! 
I'll have a look if this package might help us out: https://appsilon.github.io/shiny.fluent/reference/Overlay.html